scripts/publish_code_coverage.js

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env node
+
+// Script to retrieve total code coverage ratio from codecov.io
+// for a given commit hash, and upload it to the S3 bucket.
+
+const https = require('https');
+const zlib = require('zlib');
+const AWS = require('aws-sdk');
+const {execSync} = require('child_process');
+
+const args = process.argv.slice(2);
```

Comment:
nit: https://www.npmjs.com/package/args ?

Reply:
I tried to minimize npm module dependencies :)